### PR TITLE
added type definitions for NPM package module-alias

### PR DESCRIPTION
**Pull request by kevinramharak**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/module-alias/module-alias-tests.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import * as moduleAlias from 'module-alias';
```
Comment (DanielRosenwasser) [inline types/module-alias/module-alias-tests.ts]: Can you switch this to an `import moduleAlias = require("module-alias")` and add a test using this as a function?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/module-alias/module-alias-tests.ts (hunk 1 above)
Comment (KevinRamharak) [reply types/module-alias/module-alias-tests.ts]: commited requested changes in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/25911/commits/6891a455e9b34b7668ed1429604cad59fe485c0b

**Comment by DanielRosenwasser** [comment]
Just as a heads up, your commits don't seem to be associated with your GitHub account. While this isn't technically a problem, you might care if you want more appropriate attribution. You can either make sure your GitHub account is associated with the email address you're using for your commits, or rebase and amend your commits to fix the author name and email.

If you don't care about any of this stuff, I can always just pull in the changes.

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by KevinRamharak** [comment]
Well i don't really care for these commits, but thanks for mentioning i will fix it for my future projects

**Comment by KevinRamharak** [comment]
I noticed it was my old email address that was linked to those commits, so i just linked that email to my github account. It seems to be fine now

Also sorry I accidentally closed the pull request on my previous comment.

**Pull request merged**